Doppler.BigQueryMicroservice/Controllers/BigQueryController.cs

```diff
@@ -49,6 +65,28 @@ public async Task<ActionResult> SaveAllowedEmails(string accountName, AllowedEma
             var result = await _userAccessByUserRepository.MergeEmailsAsync(user.IdUser, model.Emails);
             return Ok(result);
         }
-    }
 
+        /// <summary>
+        /// Notify allowed users from doppler raly
+        /// </summary>
+        /// <param name="accountName">default account name from doppler</param>
+        /// <param name="emails">List of emails to notify</param>
+        /// <returns>true if not erros</returns>
+        [HttpPost("/big-query/{accountName}/notify-emails")]
+        public async Task<ActionResult> NotifyAllowedEmails(string accountName, EmailNotifier model)
+        {
+            try
+            {
+                object emailData = new { accountName = accountName };
+                var template = _emailSettings.Value.BigQueryInvitationTemplateId[model.Languaje.ToLower()];
+                var result = await _emailSender.SendWithTemplateAsync(template, emailData, model.Emails);
+                return Ok(result);
+            }
+            catch (System.Exception ex)
+            {
+                _logger.LogError(ex, $"Error in method NotifyAllowedEmails, controller  {nameof(BigQueryController)}");
```

Comment:
In general, we try to use [structured logging](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-5.0#log-message-template-1):

```suggestion
                _logger.LogError(ex, "Error in method NotifyAllowedEmails, controller {controller}", nameof(BigQueryController));
```

Please read about structured logging [here](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-5.0#log-message-template-1) and [here](https://softwareengineering.stackexchange.com/questions/312197/benefits-of-structured-logging-vs-basic-logging).

But in this case, I think that this information is redundant because it is in the log line and in the stack trace, so:

```suggestion
                _logger.LogError(ex, "Error in notifying allowed emails");
```

Reply:
suggest accepted and applied